src/formatters/html.js

```diff
@@ -20,6 +20,10 @@ function htmlEscape(text) {
   return html;
 }
 
+HtmlFormatter.prototype.prepareContext = function(context) {
+  BaseFormatter.prototype.prepareContext.call(this, context);
+};
```

Comment:
isn't this identical to current behavior? (if the inheriting method doesn't do anything else)